grammars/go.cson

```diff
@@ -162,19 +169,37 @@
         "match": "(\\+|&|\\+=|&=|&&|==|!=|\\-|\\||\\-=|\\|=|\\|\\||<|<=|\\*|\\^|\\*=|\\^=|<\\-|>|>=|/|<<|/=|<<=|\\+\\+|=|:=|;|%|>>|%=|>>=|\\-\\-|!|\\.\\.\\.|&\\^|&\\^=)"
       }
       {
-        "name": "keyword.operator.bracket.go",
-        "match": "(\\(|\\)|\\[|\\]|\\{|\\})"
+        'match': '\\{|\\}'
+        'name': 'meta.brace.curly.go'
       }
       {
-        "name": "keyword.operator.punctuation.go",
-        "match": "(\\.|,|:)"
+        'match': '\\(|\\)'
+        'name': 'meta.brace.round.go'
+      }
+      {
+        'match': '\\[|\\]'
+        'name': 'meta.brace.square.go'
       }
     ]
   'variables':
     'patterns': [
       {
-        'name': 'variable.go'
-        'match': '\\b[\\w_][\\w\\d_]*\\b',
+        'match': '\\b[\\w_][\\w\\d_]*\\b'
```

Comment:
That's true, but those (type's) it seem to be hard to scope them consistently as also discussed on Slack and here:  https://github.com/atom/language-go/pull/49

For the package name it should be no problem.